doc/source/coregistration.md

```diff
@@ -305,20 +311,17 @@ _ = ax[1].set_yticklabels([])
 plt.tight_layout()
 ```
 
-(icp)=
-### Iterative closest point
+(lzd)=
+### Least Z-difference
 
-{class}`xdem.coreg.ICP`
+{class}`xdem.coreg.LZD`
 
 - **Performs:** Rigid transform transformation (3D translation + 3D rotation).
```

Comment:
Should "Rigid transform transformation" be "Rigid transformation"? It appears several times.